debezium-connector-postgres/src/main/java/io/debezium/connector/postgresql/TypeRegistry.java

```diff
@@ -235,6 +243,14 @@ public int hstoreOid(){
         return hstoreOid;
     }
 
+    /**
+    *
+    * @return OID for array of {@code HSTORE} type of this PostgreSQL instance
+    */
+   public int hstoreArrayOid(){
```

Comment:
```suggestion
   public int hstoreArrayOid() {
```

Reply:
Fixed. Also filed https://issues.jboss.org/browse/DBZ-1341 for adding this to CS.